rustic-rustfmt.el

```diff
@@ -261,10 +281,7 @@ This operation requires a nightly version of rustfmt.
 
 ;;;###autoload
 (defun rustic-format-buffer ()
-  "Format the current buffer using rustfmt.
-
-Provide optional argument NO-STDIN for `rustic-before-save-hook' since there
-were issues when using stdin for formatting."
```

Comment:
It seems the line wasn't removed when we changed this function in the past. Thanks for fixing it.